### ISSUE TITLE
[BUG] Runs Slow on First Run

### ISSUE THREAD

**Issue by roaramburu** [BlazingDB/blazingsql#140]
**Describe the bug**
All queries run slower on the first launch of BlazingSQL. If I launch a python Kernel with BlazingSQL all queries and create_table statements run slower. If I restart the kernel and launch again BlazingSQL gets dramatically faster.

**Steps/Code to reproduce bug**
I launched a new GCP server with CUDA 10.0 installed.
I installed miniconda and then installed bsql with dask-cuda and jupyterlab:
 ```bash
conda install -c blazingsql-nightly/label/cuda10.0 -c blazingsql-nightly -c rapidsai-nightly -c conda-forge -c defaults blazingsql
conda install -c rapidsai-nightly dask-cuda
```
I launched a jupyterlab and executed the following code:
```python
from blazingsql import BlazingContext
from dask_cuda import LocalCUDACluster
from dask.distributed import Client
cluster = LocalCUDACluster()
client = Client(cluster)
bc = BlazingContext(dask_client=client, network_interface='lo')
files = ['/home/rodrigo/tpch_sf100/lineitem/0_0_' + str(num) + '.parquet' for num in range(0,72)]
bc.create_table('lineitem',files)
```
The above code will run dramatically faster if I restart the kernel and run again. Something seems to be making the first launched kernel to run slower than it should.

**Expected behavior**
It should run equivalently as fast with the first launch or the *nth* launch.

**Environment overview (please complete the following information)**
 - n1-standard-8 w/ 4 Tesla T4 GPUs, CUDA 10.0, Ubuntu 16.04
 - Method of BSQL install: [conda, Docker, or from source]
   - conda blazingsql-nightly

**Additional context**
![image](https://user-images.githubusercontent.com/5353722/69587659-06674480-0f9b-11ea-92f1-a643482fc014.png)

**Comment by aucahuasi** [comment]
Thanks @roaramburu yeah it seems we have an issue somewhere, here @rommelDB will help use with this one +1

**Comment by aucahuasi** [comment]
@rommelDB when you reproduce this please do tests on jupyterlab as well (jupyterlab=1.1.4)

**Comment by rommelDB** [comment]
Hey @roaramburu, unfortunately, I couldn't reproduce the issue on the same scale. For example, on my local machine in the first run, the attached script took 3.98s, but in the second run took 1.35s.

I performed further experiments, line by line, and apparently, the Jupyter Notebook does not re-execute some code cells that were previously executed (mainly the importing statements). This behavior would explain the difference of a few seconds.

Therefore, the time difference that I observed, might not be a bug on our side, but an optimization of the execution environment, since it has been possible to verify that the execution time of the same script on a terminal is consistent between the first and subsequent executions.

cc @aucahuasi
